Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Unreleased
 
+# v0.14.0
+- [fixed] Fixed compilation with in C99 and C++11 modes without GNU extensions.
```

Comment:
"with in" ==> typo? i'd go with "compilation in C99 and C++11 modes" personally

Reply:
https://www.youtube.com/watch?v=cnaeIAEp2pU

Reply:
lol, i mean, it could've been "with" or "in" or "within" — you were just covering all of the potential prepositions. +10 points for being thorough!